### ISSUE TITLE
Azure Load Test runs for a minute and then terminates (Help needed)

### ISSUE THREAD

**Issue by johnAgilisys** [microsoft/azure-load-testing#91]
**Describe the bug**
Azure Load Test runs for a minute and then terminates

**To Reproduce**
1. Go to Azure Portal
2. Click on 'Create a test plan and run the test '

**Expected behavior**
The test should run for the duration specified in the Jmeter (Thread Group) configuration (240Seconds)

**Screenshots**
![image](https://user-images.githubusercontent.com/102304772/177964637-3fc45d82-3410-4dc7-9130-761ae4aa74c4.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser [chrome]

**Additional context**

AB#1568023

**Comment by johnAgilisys** [comment]
@ninallam - any inputs into this..not sure if this is an existing feature

**Comment by ninallam** [comment]
@johnAgilisys Yes, the test should run for the duration specified in the JMX. Can you check if the error rate was very high (>90%) for the 1 min that the test ran for?

**Comment by johnAgilisys** [comment]
@ninallam - Thanks for the quick response. No errors. Screenshot attached. My test runid is testRunId/ce131c17-8f39-4e15-a0a6-6eb7e8b0c58a

![image](https://user-images.githubusercontent.com/102304772/177994302-ea68200c-1fd8-4ae0-9fee-b59e98500adf.png)

**Comment by johnAgilisys** [comment]
@ninallam - Any thoughts on what the issue could be please.

**Comment by ninallam** [comment]
@johnAgilisys are you still facing this issue?

**Comment by johnAgilisys** [comment]
@ninallam Thanks for the response. Yes still no change.. 
![image](https://user-images.githubusercontent.com/102304772/205271397-3c1829d9-0dff-41ef-a192-f6aa0d15f444.png)
![image](https://user-images.githubusercontent.com/102304772/205271932-2b5eb944-a848-4603-a643-2bf0bb473f9b.png)

**Comment by ninallam** [comment]
@johnAgilisys Thanks for confirming can you please provide the test run ID?

**Comment by johnAgilisys** [comment]
@ninallam no worries. Assuming you are after this [/testRunId/aaaa26f1-4201-4db1-b889-fc9a400ab885]

**Comment by ninallam** [comment]
@johnAgilisys We have enabled few more diagnostics recently which would help us get more idea on what is happening here. Can you please re-run and check if this is still an issue. If yes, can you please share the test run ID?

**Comment by johnAgilisys** [comment]
@ninallam [testRunId/331c5a14-3306-4b95-a77a-29d3d584c671]

![image](https://user-images.githubusercontent.com/102304772/215440423-68cbf620-4f4b-478d-9dac-b5cd43e7be75.png)

**Comment by ninallam** [comment]
@johnAgilisys Are you still facing this issue?
